### DISCUSSION TITLE
Additive manufacturing  simulation in Radioss?

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#2765] [Q&A]
Hello Radioss exports.

I wondering there is any possibility to do additive manufacturing simulation in Radioss? Any example?

Thanks

**Comment by MarianBulla** [comment]
Hello @li2qn ,

yes, it is possible to do additive manufacturing simulation in Radioss.
You just need to use the /ACTIV keyword, and activate vs. time your elements:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/491a5341-365a-4101-957c-db68f900136d)
[activation5_0000.txt](https://github.com/user-attachments/files/15965427/activation5_0000.txt)
(Please rename the *.txt file into *.rad)

This is a very simple example, just showing one usecase.
You can add different tempeartures and other properties to the activated elements, once they appear.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks so much, This is very helpfully. How about the Modeling metallurgical phase transformations (phase transformation process)? does Radioss also have the capacity?  Something like attached? Thanks again for your info.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/9723469b-c2bb-4876-b37b-ed8637908524)

**Reply by MarianBulla** [reply]
Yes, it is possible, but only with the ALE or SPH approach.
Using lagrangian aproach, I'm not sure if it makes sense.
However, you can start shortly after solidification, where the shape is set. And continue concidering the cooling phase with all its physical effects. 
This should work with the /HEAT/MAT/  and the /THERMSTRESS options.

Using a material model, which considers the temperature effects (we have many of them, which can do it).

Hope it helps you ?

Thanks and best regards,
Marian
